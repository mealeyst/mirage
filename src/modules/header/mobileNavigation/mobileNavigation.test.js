import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { MobileNavigation } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) MobileNavigation', () => {
  const createMobileNavigation = (props) => {
    return shallowWithTheme(<MobileNavigation {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createMobileNavigation())
    .toMatchSnapshot()
  })
})
