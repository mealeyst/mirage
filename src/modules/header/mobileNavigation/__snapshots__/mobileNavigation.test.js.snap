// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) MobileNavigation matching the snapshot 1`] = `
.c0 .screenReader {
  font-size: 0;
}

.c0 .c1 {
  max-width: 22.4rem;
}

.c0 ul ul {
  margin-bottom: 2rem;
}

.c0 ul ul ul {
  margin-bottom: 0;
}

<BaseMobileNavigation
  className="c0"
  drawerPosition="fixed"
  homepageUrl="https://rocketsofawesome.com"
  outfitNav="true"
  showBlog=""
/>
`;
